docs/src/10_quality_requirements.adoc

```diff
@@ -2,72 +2,32 @@ ifndef::imagesdir[:imagesdir: ../images]
-
-For architects, two kinds of scenarios are important:
-
-* Usage scenarios (also called application scenarios or use case scenarios) describe the system’s runtime reaction to a certain stimulus. This also includes scenarios that describe the system’s efficiency or performance. Example: The system reacts to a user’s request within one second.
-* Change scenarios describe a modification of the system or of its immediate environment. Example: Additional functionality is implemented or requirements for a quality attribute change.
-
-.Motivation
-Scenarios make quality requirements concrete and allow to
-more easily measure or decide whether they are fulfilled.
-
-Especially when you want to assess your architecture using methods like
-ATAM you need to describe your quality goals (from section 1.2)
-more precisely down to a level of scenarios that can be discussed and evaluated.
-
-.Form
-Tabular or free form text.
-****
+To obtain a measurable system response to stimulus corresponding to the various quality branches outlined in the mindmap, we will use quality scenarios. Scenarios make quality requirements concrete and allow to more easily measure or decide whether they are fulfilled.
+
+==== Usage Scenarios
+[options="header",cols="1,3,1"]
+|===
+|Quality attribute|Scenario|Priority
+| Functional suitability | Users should be able to register, log in, play the quiz, and access historical data without encountering errors or glitches. | High, Medium
+| Security | User data must be securely handled. Robust authentication mechanisms should be in place for user registration and login. API access points for user information and generated questions must be secured with proper authorization. | High, High 
+| Reliability | The system should reliably generate accurate and diverse questions from Wikidata. User registrations, logins, and game data storage should be handled without errors. | High, Medium
+| Availability | The system must be available 99.99% of the time when a user attempts to access it. | High, High
+| Performance efficiency | The system must deliver optimal performance, ensuring responsive interactions for users. It should efficiently generate questions from Wikidata and handle real-time gameplay with up to 20 concurrent users. | High, High
+| Usability | The system must provide a user-friendly interface, allowing users to register, log in, and play the game with a learning time of less than 4 hours. | High, Medium
+| Compatibility | The system must be compatible with various web browsers and devices, providing a seamless experience for users regardless of their choice of platform. It should be well-optimized for different screen sizes and functionalities. | High, Medium
+| Transferability | The system must allow for easy transfer of user data and game-related information through its APIs. | Medium, High
+| Testability | The unit tests must have at least 90% coverage. | High, Medium
```

Comment:
I know this is just a random value but dont put 90% coverage as a testability measure, coverage does not ensure a system is properly tested. For now lets keep it here but maybe reduce it to 75% to make it easier to achieve please.

Reply:
Code coverage might seem easy to increase, but based on my experience from last year, 75% is much more realistic figure than 90%. There is also the bit regarding availability: it does not really depend on us, but rather on our cloud provider. What we can do on our end is make sure the system is as solid as possible as to limit the as much as possible the chances of it crashing.